chart/kubeapps/templates/mongodb-secret-cleanup.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ template "kubeapps.mongodb-secret-cleanup.fullname" . }}
+  annotations:
+    helm.sh/hook: post-delete
+    helm.sh/hook-delete-policy: hook-succeeded
+  labels:
+    app: {{ template "kubeapps.mongodb-secret-cleanup.fullname" . }}
+    chart: {{ template "kubeapps.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  template:
+    metadata:
+      labels:
+        app: {{ template "kubeapps.mongodb-secret-cleanup.fullname" . }}
+        release: {{ .Release.Name }}
+    spec:
+      containers:
+      - name: kubectl
+        image: {{ template "kubeapps.image" .Values.hooks.image }}
+        command:
+        - kubectl
```

Comment:
what happens if the secret somehow doesn't exist, this will fail? Should we account for that?

Reply:
The same than the bootstrap. The job execution will fail but it will be cleanup anyway.